src/ILCompiler.WebAssembly/src/Compiler/DependencyAnalysis/WebAssemblyCodegenNodeFactory.cs

```diff
@@ -10,10 +10,16 @@ namespace ILCompiler.DependencyAnalysis
 {
     public sealed class WebAssemblyCodegenNodeFactory : NodeFactory
     {
+        private NodeCache<MethodKey, WebAssemblyVTableSlotNode> _vTableSlotNodes;
```

Comment:
This can just use `MethodDesc` as the key. `MethodKey` is used when the method can be an unboxing stub. It can't in this case.